aw_watcher_window/macos_jxa.py

```diff
@@ -44,9 +49,109 @@ def compileScript():
 
     return script
 
+def wrapped_function(condition, result_queue: multiprocessing.Queue, function_reference: t.Callable) -> None:
+    # this is run in a forked process, which wipes all logging configuration
```

Comment:
Forking wipes logging config? :O

Reply:
Seems to. I think it has something to do with the logging config tied to file handles which aren't valid in the forked process? Unsure, was hoping you/someone else understood this more.

Reply:
File handle sounds like a likely culprit. But, just to make sure, "wipes *all* logging configuration" isn't true, is it?